code/modules/jobs/job_types/_job.dm

```diff
@@ -87,6 +91,12 @@
 		for(var/t in mind_traits)
 			ADD_TRAIT(H.mind, t, JOB_TRAIT)
 
+	var/obj/item/organ/liver/liver = H.getorganslot(ORGAN_SLOT_LIVER)
+
+	if(liver && liver_traits)
+		for(var/t in liver_traits)
```

Comment:
```suggestion
	if(liver)
		for(var/t in liver_traits)
```
`for(var in null)` does what one would expect, no need to check if the list exists before